### ISSUE TITLE
Question about paper: validation loss of HMDB51

### ISSUE THREAD

**Issue by martinetoering** [kenshohara/3D-ResNets-PyTorch#144]
Hi,

First of all, great work! 

I had a question about the method described in your paper from 2018. Figure 4 shows the training and validation losses for HMDB51 and other datasets. However, HMDB51 only has train and test as far as I'm aware. My question is how the validation data was obtained; did you create a validation set for these experiments or is there a validation for HMDB51 that I have overlooked?

Thanks!
Martine
